csharp/src/Drivers/Apache/Spark/SparkConnection.cs

```diff
@@ -32,14 +32,12 @@
 using Apache.Arrow.Ipc;
 using Apache.Arrow.Types;
 using Apache.Hive.Service.Rpc.Thrift;
-using Thrift;
-using Thrift.Protocol;
 
 namespace Apache.Arrow.Adbc.Drivers.Apache.Spark
 {
-    public class SparkConnection : HiveServer2Connection
+    internal abstract class SparkConnection : HiveServer2Connection
```

Comment:
Now an abstract class to be used for common properties/methods for specific Spark variants.